lib/langchain/embeddings.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Langchain
+  module Config
+    EMBEDDING_SIZES = {
```

Comment:
Do we intend to put ALL of the embedding sizes across the different LLM providers in 1 constant?

Reply:
I wasn't too sure about this constant and I don't know what OpenAI's evolution strategy is, but I intended to be able to have these constants available to the class if needed, as in the case of the test specification. I think this OpenAI's approach is only for embeddings

Reply:
Everybody is having queries/concerns about this changes -> : https://community.openai.com/t/how-to-deal-with-different-vector-dimensions-for-embeddings-and-search-with-pgvector/602141

Reply:
I added a screenshot of the current behavior and the fixed behavior.

Reply:
I ran all my tests yesterday  night and all passed, not sure why currently some are broken now